tools/recetox_aplcms/macros.xml

```diff
@@ -146,6 +145,25 @@
                    help="The minimum number of raw data points to be considered as a true feature." />
         </section>
     </xml>
+    <xml name="multibatch_processing">
+        <section name="multibatch_processing" title="Multibatch processing">
+            <param name="min_within_batch_prop_detect" type="float" min="0" max="1" value="0.1"
+                   label="minimum_batchwise_prop_detect"
+                   help="The minimum proportion of samples in a batch that must have a given feature. The features that are less abundant than the value won't be extracted." />
+            <param name="min_within_batch_prop_report" type="float" min="0" max="1" value="0.5"
+                   label="minimum_batchwise_prop_report"
+                   help="The minimum proportion of samples in a batch that must have a given feature. The features that are less abundant than the value won't be reported in the final output." />

```

Comment:
What does `reported` actually mean in this case? I think this is rather confusing - having a combined parameter with the detection frequency to extract the feature might make more sense, since the distinction between what is included in the table and what is reported is a bit confusing.

Reply:
You're right. It does make more sense to have one parameter for that. Especially because when `min_within_batch_prop_detect` is lower than `min_within_batch_prop_report` (as by default) only the former has practical significance. Removed `min_within_batch_prop_report` in 3e752f9.